openpype/hosts/blender/plugins/publish/extract_blend.py

```diff
@@ -23,24 +50,28 @@ def process(self, instance):
         # Perform extraction
         self.log.info("Performing extraction..")
 
+        plugin.deselect_all()
+
         data_blocks = set()
+        objects = set()
 
         for obj in instance:
```

Comment:
Could you please add a comment here? I don't get explicitly the purpose of this loop...

Reply:
done